frontend/src/App.tsx

```diff
@@ -214,6 +222,12 @@ function appStateReducer(state: CoveyAppState, update: CoveyAppUpdate): CoveyApp
       closedInstructions = true;
       nextState.gameStarted = true;
       break;
+    case 'openLeaderboard':
+      nextState.showLeaderboard = true;
+      break;
+    case 'closeLeaderboard':
+      nextState.showLeaderboard = false;
+      break;
```

Comment:
minor suggestion: could've done one `toggleLeaderboard` action that takes in a boolean, but im fine with this too

Reply:
this is cleaner -- updated